source/includes/_opt_out_js.md

```diff
@@ -0,0 +1,15 @@
+# Opt out from Surveys
+
+```js
+window.wootricSettings = {
+  email:'nps@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+  created_at: 1234567890, // TODO: The current logged in user's sign-up date as a Unix timestamp.
+  account_token: 'NPS-xxxxxxx',
+  show_opt_out: true
+};
+```
+
+You can offer a way to opt-out from surveys from within the modal. To do so, you must enable the ```show_opt_out``` flag.
```

Comment:
You can offer `users` a way to opt-out...

Reply:
Perfect, thanks!